src/loader.py

```
@@ -84,22 +93,102 @@ def load_file_source(entry: Metadata, executor: Executor, params: Params) -> str
+
+    # Initialize data without the timestamp column if it will be used as index
+    init_columns = [col for col in column_names if col != tstamp_col] if has_tstamp else column_names
+    data = pd.DataFrame(columns=init_columns)
+    if has_tstamp:
+        # Set the index name on the empty DataFrame so it's preserved during concat
+        data.index.name = tstamp_col
+
+    for fname in fnames:
+        try:
+            df = pd.read_csv(fname, **args)
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)